pulp_smash/tests/pulp3/pulpcore/api_v3/test_api_docs.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+"""Test related to the api docs page."""
+import unittest
+
+from requests.exceptions import HTTPError
+
+from pulp_smash import api, config, selectors, utils
+from pulp_smash.tests.pulp3.constants import API_DOCS_PATH
+from pulp_smash.tests.pulp3.pulpcore.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class ApiDocsTestCase(unittest.TestCase, utils.SmokeTest):
+    """Test whether API auto generated docs are available.
+
+    This test targets the following issues:
+
+    * `Pulp #3552 <https://pulp.plan.io/issues/3552>`_
+    * `Pulp Smash #893 <https://github.com/PulpQE/pulp-smash/issues/893>`_
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variables."""
+        cls.cfg = config.get_config()
```

Comment:
This variable is only used in this method. Make it a local variable, not a class variable. It can be made a class variable in the future if it's needed more broadly.